frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineSuccessRatioDonut.tsx

```diff
@@ -99,10 +99,16 @@ const PipelineSuccessRatioDonut: React.FC<PipelineMetricsGraphProps> = ({
   }
   const successValue = _.find(finalArray, { x: 'success' })?.['count'] ?? 0;
   const successData = _.sortBy(finalArray, 'sortOrder');
+  const OVERLAP = 20;
   return (
     <Grid hasGutter>
       <GridItem xl2={3} xl={3} lg={3} md={3} sm={3}>
-        <div style={{ height: DEFAULT_CHART_HEIGHT }}>
+        <div
+          style={{
+            height: DEFAULT_CHART_HEIGHT,
+            marginTop: OVERLAP * -1,
```

Comment:
Added here based on the implementation to solve the [tooltip issue](  https://github.com/openshift/console/pull/8062)

Reply:
Interesting. Well that's bad on my part. I wonder if the solution is actually to add padding to inverse the margin rather than to include more margin offsets 🤔 

We needed space for the tooltip due to poor portal-ing by PF (or Victory).